src/extensions/geo-export/mesh-exporter.ts

```diff
@@ -349,15 +365,38 @@ export abstract class MeshExporter<D extends RenderObjectExportData> implements
             drawCount = values.drawCount.ref.value;
         }
 
-        await this.addMeshWithColors({ mesh: { vertices: aPosition, normals: aNormal, indices, groups: aGroup, vertexCount, drawCount }, meshes: undefined, values, isGeoTexture: false, webgl, ctx });
+        await this.addMeshWithColors({ mesh: { vertices: aPosition, normals: aNormal, indices, groups: aGroup, vertexCount, drawCount }, meshes: undefined, values, isGeoTexture: false, mode: 'triangles', webgl, ctx });
     }
 
     private async addLines(values: LinesValues, webgl: WebGLContext, ctx: RuntimeContext) {
-        // TODO
+        const aStart = values.aStart.ref.value;
+        const aEnd = values.aEnd.ref.value;
+        const aGroup = values.aGroup.ref.value;
+        const vertexCount = (values.uVertexCount.ref.value / 4) * 2;
+        const drawCount = values.drawCount.ref.value / (2 * 3);
+
+        const n = (vertexCount / 2);
+        const vertices = new Float32Array(n * 2 * 3);
+        for (let i = 0; i < n; ++i) {
+            vertices[i * 6] = aStart[i * 4 * 3];
```

Comment:
Why the `4 *`?

Reply:
we use two triangles made with 4 vertices to draw wide (more than a pixel) lines because webgl implementers don't support wide lines using the lines primitive

Reply:
Could this be achieved by a single zero area triangle as well? ...two vertices are the endpoints with the last point being in the middle and having a shader similar to how we do outlines?